docs/quickstarts/ratify-on-azure.md

```diff
@@ -127,7 +127,8 @@ Configure user-assigned managed identity and enable `AcrPull` role to the worklo
     --object-id ${IDENTITY_OBJECT_ID}
     ```
 
-## Deploy Gatekeeper and Ratify on AKS 
+## Deploy Gatekeeper and Ratify on AKS
+### When Azure Policy Addon is not enabled
```

Comment:
We probably need some context around What is a Azure policy addon, when would the customer need it.etc . audience maybe not be familiar with azure policy addon